find_number.rb

```diff
@@ -0,0 +1,39 @@
+is_session_active = true
+is_game_active = true
+
+while is_session_active
+  num_to_guess = rand(1..100)
+
+  while is_game_active
+
+    puts "Guess a number between 1 and 100"
+    guess = gets.chomp
```

Comment:
❕ Instead of repeatedly converting the `guess` to an integer, why not save it as an integer?

```suggestion
    guess = gets.chomp.to_i
```